nucliadb/src/nucliadb/search/api/v1/find.py

```diff
@@ -192,9 +193,12 @@ async def _find_endpoint(
 ) -> Union[KnowledgeboxFindResults, HTTPClientError]:
     try:
         maybe_log_request_payload(kbid, "/find", item)
-        results, incomplete, _ = await find(kbid, item, x_ndb_client, x_nucliadb_user, x_forwarded_for)
-        response.status_code = 206 if incomplete else 200
-        return results
+        with cache.request_caches():
```

Comment:
Is it really necessary to be explicit about caches on each search endpoint? It feels is repeating code and we always want to use caches.

Reply:
will change it to a middleware!